yarn.lock

```diff
@@ -1862,6 +1862,11 @@
   resolved "https://registry.yarnpkg.com/@types/node/-/node-11.11.3.tgz#7c6b0f8eaf16ae530795de2ad1b85d34bf2f5c58"
   integrity sha512-wp6IOGu1lxsfnrD+5mX6qwSwWuqsdkKKxTN4aQc4wByHAKZJf9/D4KXPQ1POUjEbnCP5LMggB0OEFNY9OTsMqg==
 
+"@types/node@^11.13.5":
```

Comment:
Theses changes are the result of a new fresh install of the `public-cloud` branch.

```sh
$ node --version && yarn --version
v10.13.0
1.15.2
```